Resources/Prototypes/_Goobstation/Changeling/Entities/Objects/Weapons/Melee/changeling_armblade.yml

```
@@ -0,0 +1,50 @@
+- type: entity
+  parent: ArmBlade
+  id: ArmBladeChangeling
+  suffix: Unremoveable
+  components:
+  - type: Sharp
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)